wptherml/materials.py

```diff
@@ -1643,3 +1643,71 @@ def _read_Atmospheric_Transmissivity(self):
         _spline_value = _atrans_spline(7.1034e-6)
         assert np.isclose(_expected_value, _spline_value)
         return _atrans_spline(self.wavelength_array)
+    
+    def _EQE_spectral_response(self):
+        """ 
+        Will compute the spectral response function using tabulated EQE values for user input thickness
+            from 
+
+            "Optical Properties and Modeling of 2D Perovskite Solar Cells",
+            Bin Liu, Chan Myae Myae Soe, Constantinos C. Stoumpos, Wanyi Nie, Hsinhan Tsai, Kimin
+            Lim, Aditya D. Mohite, Mercouri G. Kanatzidis, Tobin J. Marks, Kenneth D. Singer
+            Advanced Materials, (34), 1. January 6, 2022 
+            https://doi.org/10.1002/adma.202107211
+
+            on EQE curves of Pb5 perovskite-based devices.
+
+            Using formula SR = q * EQE * \lambda / (h * c)
+
+        Attributes
+        ----------
+        psc_thickness : 
+                        User-input thickness in nm of PSC.
+        _eqe_array :
+                        Table of EQE values extrapolated from graph.
+        _sr_array : 
+                        Calculated spectral response based on formula, wavelength, tabulated values, and constants.
+        _eqe_spline :
+                        Spline best fit based on wavelength array and EQE data.
+        _sr_spline :
+                        Spline best fit based on SR array and wavelength array.
+
+        Returns
+        -------
+        None
+        
+        """
+
+        # Initialize wavelength array and variable for thickness
+        _wavelength_array = np.array([250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850]) * 1e-9
+
+        # Check for psc values
+        if self.psc_thickness_option == 110:
+            _eqe_array = np.array([0, 0, 22, 46, 58, 59, 54, 40, 24, 18, 4,	0, 0])
+            
+        if self.psc_thickness_option == 200:
```

Comment:
Remember to change the subsequent conditional statements to 'elif' statements instead of a series of 'if' statements.  Also conclude with an "else" statement that gives the matches the default value.